actions/nix-build/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "nix-build",
+  "version": "0.0.1",
```

Comment:
I think this should be `0.0.0` and once it's release it will be `0.1.0`

Reply:
Thanks for this. I wasn't sure whether 0.0.0 was a valid version.